doc/engineering-reference/src/simulation-models-encyclopedic-reference-002/heat-exchangers.tex

```diff
@@ -1129,15 +1138,57 @@ \subsubsection{Summary of Variable Short Time Step Response Factor Model}\label{
 
 \subsubsection{References}\label{references-2-006}
 
-Eskilson, P. 1987. Thermal Analysis of Heat Extraction Boreholes. Ph.D.~Thesis, Department of Mathematical Physics, University of Lund, Lund, Sweden.
+\hangindent=2em
+\hangafter=1
+\noindent Claesson, J., G. Helstr{\"o}m. 2011. Multipole method to calculate borehole thermal resistances in a borehole heat exchanger. HVAC\&R Research. 17(6), 895-911.
+
+\hangindent=2em
+\hangafter=1
+\noindent Cook, J. C. and J. D. Spitler. 2021. Faster computation of g-functions used for modeling of ground heat exchangers with reduced memory consumption. Building Simulation 2021. Bruges, Belgium, IBPSA.
+
+\hangindent=2em
+\hangafter=1
+\noindent Cimmino, M. (2015). "The effects of borehole thermal resistances and fluid flow rate on the g-functions of geo-thermal bore fields." International Journal of Heat and Mass Transfer 91: 1119-1127.
```

Comment:
Not sure what these red highlights are in Cimmino's citations. I'll see if I can get those off on my next push.

Reply:
It doesn't like the double-quote symbol. Quotes should open with two back-ticks (I forget the proper name) and close with two apostrophes. ``set up''

Reply:
Looks like that resolved it, thanks @mjwitte !!